File: 2048-BuildArrayFromPermutation/2048-BuildArrayFromPermutation.py
# Last updated: 9/3/2025, 10:47:07 AM
class Solution:
    def buildArray(self, nums: list[int]) -> list[int]:
        ans = [0] * len(nums)
        for i in range(len(nums)):
            ans[i] = nums[nums[i]]
        return ans

File: 2048-BuildArrayFromPermutation/2048-BuildArrayFromPermutation.java
// Last updated: 9/3/2025, 10:47:12 AM
class Solution {
    public int[] buildArray(int[] nums) {
        int n=nums.length;
        int [] result=new int[n];
        for(int i=0;i<n;i++){
            result[i]=nums[nums[i]];
        }
        return result;
    }
}